platformio.ini

```diff
@@ -7,3 +7,68 @@
 ;
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+include_dir = firmware/include
+lib_dir = firmware/lib
+src_dir = firmware/src
+test_dir = firmware/test
+
+[env]
+platform = ststm32
+
+# WARNING: For whatever reason, this CANNOT be the
+# disco_b_g431b_esc1 board you probably anticipate
+# it can be. This took my morning don't let it take yours.
+board = genericSTM32G431CB
+framework = arduino
+monitor_dtr = 1
+monitor_speed = 115200
+
+lib_deps =
+    # these both need to be on the dev branch, but we cannot spec a branch for deps
+    # instead we've taken the tip of the dev branches for each libs (as of 22-feb-2024)
+    https://github.com/simplefoc/Arduino-FOC-drivers.git#3a11d581d73018d3836614d674210e017da43661
```

Comment:
@napowderly while you can't do branches, it does seem like you're able to use commits!